spaghetto/spago.yaml

```diff
@@ -63,3 +64,6 @@ workspace:
       git: https://github.com/purescript/registry-dev.git
       ref: 8113deecae25410ba62c31a5842c6ebbf1c12461
       subdir: foreign
+    node-execa: 1.1.0
+    node-event-emitter: 1.0.1
+    node-human-signals: 1.0.0
```

Comment:
`node-execa` is now in the package set, but I'm not sure how to indicate that here...

Reply:
You should bump the package set to one that contains the library, in line 55.
Latest is [`11.10.0`](https://github.com/purescript/registry/blob/main/package-sets/11.10.0.json), and contains all the libraries added here.
It has `dodo-printer` too, so you can remove line 57 as well.